jax/scipy/special.py

```diff
@@ -15,6 +15,11 @@
 # Note: import <name> as <name> is required for names to be exported.
 # See PEP 484 & https://github.com/google/jax/issues/7570
 
+# These functions are exposed in both jax.nn (for backwards compatibility), and
```

Comment:
Note: I would *not* consider exposing these in `jax.nn` a deprecated behavior. I encourage users to prefer the `jax.nn` copy to avoid the `scipy` import required by `jax.scipy`.